validphys2/src/validphys/tests/test_loader.py

```diff
@@ -17,18 +17,22 @@
 #The sorted is to appease hypothesis
 dss = sorted(l.available_datasets - {"PDFEVOLTEST"})
 
+class MockCuts():
+    def __init__(self, arr):
+        self.arr = arr
+    def load(self):
+        return self.arr
+
 
 @composite
-def commodata_and_cuts(draw):
+def commondata_and_cuts(draw):
     cd = l.check_commondata(draw(sampled_from(dss)))
     ndata = cd.metadata.ndata
-    #TODO: Maybe upgrade to this
-    #https://github.com/HypothesisWorks/hypothesis/issues/1115
-    mask = sorted(draw(sets(sampled_from(range(ndata)))))
+    mask = sorted(draw(sets(sampled_from(range(ndata)), min_size=1)))
```

Comment:
Seeing a variable 6 functions removed from its input without even a comment makes me nervous.

Reply:
Could you elaborate? Didn’t get what you mean here.

Reply:
I'd like have a comment for this function to understand what it does without needing to mentally parse 5 functions. It's even an effort tu understand which function is taking `min_size=1` as an argument.

So I'd by happy either with a comment / docstring `# Creates a mask by selecting a subset of datapoints (eg. [0, 2, 4, 5]` or breaking it down with obvious names.

Otherwise it is not clear at first sight whether `mask` should be `[0, 1, 0, 0]` or `[1]`